cmd/crc/cmd/start.go

```diff
@@ -169,7 +169,7 @@ func validateStartFlags() error {
 	if err := validation.ValidateDiskSize(config.Get(crcConfig.DiskSize).AsInt()); err != nil {
 		return err
 	}
-	if err := validation.ValidateBundle(config.Get(crcConfig.Bundle).AsString(), crcConfig.GetPreset(config)); err != nil {
+	if err := validation.ValidateBundlePath(config.Get(crcConfig.Bundle).AsString(), crcConfig.GetPreset(config)); err != nil {
```

Comment:
This reverts https://github.com/code-ready/crc/commit/f148ddc8687689a97c9b7e6baac4697c17b3547b and it looks like it will make the .crcbundle file required again even if it's already uncompressed.

Reply:
@cfergeau This function actually don't run `ValidateBundlePath(bundlePath, preset)` in case bundle is already extracted, no matter which version of bundle even it is supported by crc or not and we are not able to provide early warning to the users in case different bundle in use. Either we have to add another logic here where after getting `bundlemetadata` we also check if the bundle is supported on or not and show the user warning or use it in current form. Yes it will be require `.crcbundle` to be available on `~./cache` folder even it is uncompressed.


```golang
func ValidateBundle(bundlePath string, preset crcpreset.Preset) error {
	bundleName := filepath.Base(bundlePath)
	_, err := bundle.Get(bundleName)
	if err != nil {
		return ValidateBundlePath(bundlePath, preset)
	}
	/* 'bundle' is already unpacked in ~/.crc/cache */
	return nil
}
```

Reply:
Yes more code will be needed/some logic will need to be reworked. Fixing one bug by reintroducing another does not sound like a great deal.

Reply:
@cfergeau Added that logic for `ValidateBundle` .